lib/main_common.pm

```diff
@@ -124,6 +124,7 @@ our @EXPORT = qw(
   load_extra_tests_y2uitest_gui
   load_extra_tests_kernel
   load_wicked_create_hdd
+  load_wireguard_tests
```

Comment:
Do the wireguard tests run on their own test suite? if so, I'd vouch for moving this from main_common.pm to a yaml schedule instead...

Reply:
I did that changes in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11900 - please have a look.